infrastructure/data_access/http/synthetics_repo.py

```diff
@@ -38,35 +39,46 @@ def get_mesh_test_results(self, test_id: TestID, results_lookback_seconds: int)
             if num_results == 0:
                 raise Exception("get_health_for_tests returned 0 items")
 
-            most_recent_result = response.health[num_results - 1].mesh
-            return transform_to_internal_mesh(most_recent_result)
+            most_recent_result = response.health[num_results - 1]
```

Comment:
Why not `most_recent_result = response.health[-1]`?... assuming that you want the last item in the array.

Reply:
Yes, more pythonic, agree